Teacher-App/app/src/main/java/com/example/seeds/ui/call/CallViewModel.kt

```
@@ -500,9 +506,12 @@ class CallViewModel @Inject constructor(
 
             for ((phone, el) in participantsObj.entrySet()) {
                 val p = el.asJsonObject
-                val callerState = try {
+                val rawState = try {
                     CallerState.valueOf(p.get("call_status")?.asString?.uppercase() ?: "UNDEFINED")
                 } catch (e: IllegalArgumentException) { CallerState.UNDEFINED }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)